provider/clusterrosaclassic/cluster_rosa_classic_resource.go

```diff
@@ -252,6 +252,10 @@ func (r *ClusterRosaClassicResource) Schema(ctx context.Context, req resource.Sc
 					stringplanmodifier.RequiresReplace(),
 				},
 			},
+			"worker_disk_size": schema.Int64Attribute{
```

Comment:
we should throw an error if the user tries to update it

Reply:
This option needs to be treated identically to what's being done in #228 since this setting is configuring the default MP.

Since the framework upgrade, we have removed all of the `ValueCannotBeChanged` checks. Should I look into re-introducing that?
And, what about #228-- should that be changed as well?
If so, I'd prefer to move these checks to another PR.

Reply:
I think it should also be changed in #228